tests/app/views/test_brief_response.py

```diff
@@ -0,0 +1,351 @@
+import json
+
+from hypothesis import given
+
+from ..helpers import BaseApplicationTest, JSONUpdateTestMixin
+from ... import example_listings
+
+from dmapiclient.audit import AuditTypes
+from app.models import db, Lot, Brief, BriefResponse, AuditEvent
+
+
+class BaseBriefResponseTest(BaseApplicationTest):
+    def setup(self):
+        super(BaseBriefResponseTest, self).setup()
+
+        with self.app.app_context():
+            self.setup_dummy_suppliers(2)
+            brief = Brief(
+                data=example_listings.brief_data().example(),
+                status='live', framework_id=5, lot=Lot.query.get(5)
+            )
+            db.session.add(brief)
+            db.session.commit()
+
+            self.brief_id = brief.id
+
+    def setup_dummy_brief_response(self, brief_id=None, supplier_id=0):
+        with self.app.app_context():
+            brief_response = BriefResponse(
+                data=example_listings.brief_response_data().example(),
+                supplier_id=supplier_id, brief_id=brief_id or self.brief_id
+            )
+
+            db.session.add(brief_response)
+            db.session.commit()
+
+            return brief_response.id
+
+    def create_brief_response(self, data):
+        return self.client.post(
+            '/brief-responses',
+            data=json.dumps({
+                'updated_by': 'test@example.com',
+                'briefResponses': data,
+            }),
+            content_type='application/json'
+        )
+
+    def get_brief_response(self, brief_response_id):
+        return self.client.get('/brief-responses/{}'.format(brief_response_id))
+
+    def list_brief_responses(self, **parameters):
+        return self.client.get('/brief-responses', query_string=parameters)
+
+
+class TestCreateBriefResponse(BaseBriefResponseTest, JSONUpdateTestMixin):
+    endpoint = '/brief-responses'
+    method = 'post'
+
+    @given(example_listings.brief_response_data())
+    def test_create_new_brief_response(self, live_framework, brief_response_data):
```

Comment:
I like the framework pytest fixture and the hypothesis strategy. The only thing that worries me is that we now have quite a few things involved in setting up this test. Debugging it in the future will be that much more of a headache. I'm not sure if it's a big deal, it just worries me a little.